vpr/src/device/rr_graph_view.h

```diff
@@ -379,6 +379,14 @@ class RRGraphView {
     inline const t_segment_inf& rr_segments(RRSegmentId seg_id) const {
         return rr_segments_[seg_id];
     }
+    inline size_t num_rr_segments() const {
```

Comment:
Add code comments here:

/** @brief  Return the number of rr_segments in the routing resource graph */